aspnetcore/blazor/tutorials/movie-database-app/part-1.md

```diff
@@ -0,0 +1,435 @@
+
+In VS Code, press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the app without debugging.
+
+At the **Select debugger** prompt in the **Command Palette** at the top of the VS Code UI, select **C#**. At the next prompt, select the HTTPS profile (`C#: BlazorWebAppMovies [https]`).
+
+The default browser is launched at `https://localhost:{PORT}`, which displays the app's UI. The `{PORT}` placeholder is the random port assigned to the app when the app is created. If you need to change the port due to a local port conflict, you may do so in the project's `Properties/launchSettings.json` file.
+
+Navigate the pages of the app to confirm that the app is working normally.
+
+:::zone-end
+
+:::zone pivot="cli"
+
+For information on trusting the HTTPS certificate for browsers other than Firefox, see the [HTTPS development certificate trust guidance](xref:security/enforcing-ssl#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos). When using the Firefox browser, see the [certificate trust guidance for Firefox](xref:security/enforcing-ssl#trust-the-https-certificate-with-firefox-to-prevent-sec_error_inadequate_key_usage-error) section of that article.
+
+In a command shell opened to the project's root folder, execute the [`dotnet watch`](/dotnet/core/tools/dotnet-watch) command to compile and start the app:
+
+```dotnetcli
+dotnet watch
+```
+
+The app is compiled and run. The app is launched at `http://localhost:{PORT}`, where the `{PORT}` placeholder is the random port assigned to the app when the app is created. If you need to change the port due to a local port conflict, you may do so in the project's `Properties/launchSettings.json` file.
+
+Navigate the pages of the app to confirm that the app is working normally.
+
+When an app created from the Blazor Web App project template is run with the .NET CLI, the app runs at an HTTP (insecure) endpoint because the first profile found in the app's launch settings file (`Properties/launchSettings.json`) is the HTTP (insecure) profile, which is named `http`. The HTTP profile was placed in the first position to ease the transition of adopting SSL/HTTPS security for Linux and macOS users.
+
+One approach for running the app with SSL/HTTPS is to pass the [`-lp`|`--launch-profile` option](/dotnet/core/tools/dotnet-run#options) with the `https` profile name to the `dotnet watch` command:
+
+```dotnetcli
+dotnet watch -lp https
+```
+
+A more robust approach is to simply move the `https` profile above the `http` profile in the `Properties/launchSettings.json` file and save the change. After changing the profile order in the file, the `dotnet watch` command always uses the `https` profile by default.
+
```

Comment:
Ok ... ***but*** I'm 😨 again. It's scary to remove this guidance (and the other HTTPS guidance). Newbies are confused by these HTTPS gotchas 😈, and then they open issues when they get confused. I sought to write the tutorial in a way that would ward off the confusion ***and issues*** 💰.

Reply:
The decision to switch back to http by default for CLI scenarios was deliberate. If we're getting negative feedback about that we should surface it to the product team for potential solutions instead of trying to work around it in the docs.